verl/trainer/config/ppo_megatron_trainer.yaml

```diff
@@ -143,3 +143,4 @@ trainer:
   critic_warmup: 0
   default_hdfs_dir: ~/experiments/gsm8k/ppo/${trainer.experiment_name}
   default_local_dir: checkpoints/${trainer.project_name}/${trainer.experiment_name}
+  val_before_train: True
```

Comment:
Would you mind also updating the docs in https://github.com/volcengine/verl/blob/main/docs/examples/config.rst for val_before_train, and the forward batch size option added last time?

Reply:
Sure!

Reply:
Done